src/plugins/content_management/server/event_stream/memory/util.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const clone = (x: unknown) => JSON.parse(JSON.stringify(x));
```

Comment:
Yes, for a serious *clone* function that could be something that is added. But how this one is used, it only clones JSON values; I'll keep like this for now as I don't expect it will be used in that in-memory client for anything other than JSON cloning.